includes/classes/Pay/Helper/Transaction.php

```diff
@@ -111,7 +111,7 @@ public static function processTransaction($transactionId, $status = null)
             $orderStatus = $order->get_status();
         }
 
-        if ($orderStatus == 'complete' || $orderStatus == 'processing') {
+        if ( in_array($orderStatus, array('completed','processing','shipped')) ) {

```

Comment:
"shipped" seems to be a custom order status. We can't implement a custom status into our plugin.